src/filetype-plugin.ts

```diff
@@ -0,0 +1,88 @@
+import { Plugin, PluginBuild } from 'esbuild';
```

Comment:
I kinda would like to have a small description about this magic. As there is no file created after this plugin has build which is something new and something we never really did before.

Reply:
Added